handlers/handlers_test.go

```diff
@@ -411,13 +411,13 @@ func TestUnitGetCategoriesTypesCountFailure(t *testing.T) {
 		}
 
 		mockedSearchClient := &SearchClientMock{
-			GetSearchFunc: func(ctx context.Context, query url.Values) (searchC.Response, error) {
+			GetSearchFunc: func(ctx context.Context, userAuthToken, serviceAuthToken, collectionID string, query url.Values) (searchC.Response, error) {
 				return searchC.Response{}, errs.ErrInternalServer
 			},
 		}
 
 		Convey("When getCategoriesTypesCount is called", func() {
-			categories, err := getCategoriesTypesCount(ctx, apiQuery, mockedSearchClient)
+			categories, err := getCategoriesTypesCount(ctx, accessToken, collectionID, apiQuery, mockedSearchClient)
```

Comment:
We should update each test that has been affect by this change and add a check to verify the client is being called with the `accessToken`  and `collectionID`.

```
So(mockedSearchClient.GetSearchFuncCalls[0].userAuthToken, ShouldEqual, expectedToken)
So(mockedSearchClient.GetSearchFuncCalls[0].collectionID, ShouldEqual, expectedID)
```

This should hopefully by copy past into the appropriate test cases.